aem-authoring-toolkit-plugin/src/main/java/com/exadel/aem/toolkit/core/util/PluginXmlUtility.java

```diff
@@ -625,4 +625,15 @@ private void appendAcsCommonsList(Element element, String path, String resourceT
                 resourceType.isEmpty() ? ResourceTypes.ACS_LIST : resourceType);
         element.appendChild(dataSourceElement);
     }
+
+    /**
+     * Delete the current {@code Element} node and returns a parent {@code Element} node
+     * @param element Element to be deleted
+     * @return Parent element
+     */
+    public Element removeElement(Element element) {
```

Comment:
Method name is also not quite clear. Basically, in most programming languages, if "delete from collection" method has something to return, this is the removed element. I would name this like "reduceToParent" or... just did that manipulation  withing the caller routine